README.md

```diff
@@ -1,3 +1,23 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Initial setup info
+
+To run the code you will need to create 2 .env files in the "be-nc-news" directory
+
+1) The first file will need to be named ".env.development". Its contents should point towards the development database:
+
+`PGDATABASE="<name_of_development_database_here_"`
+
+2) The second file should be named ".env.test". Its contents should should point towards the testing database:
+
+`PGDATABASE="<name_of_testing_database_here>"`
+
+3) Finally make sure to run a setup.sql file with the following code:
```

Comment:
the setup.sql file already has the database names so we could just instruct the user to use those